crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -161,6 +181,10 @@ pub struct SuiTransactionResponse {
     pub effects: Option<SuiTransactionEffects>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub events: Option<SuiTransactionEvents>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub object_changes: Option<Vec<ObjectChange>>,
```

Comment:
wondering wdyt of also storing input objects? @666lcz? I kinda feel it convenient both for JSON RPC read and indexer.

Reply:
we can store this in indexer but I don't think we should store this in full node, these can be created on the indexer side.

Reply:
the tradeoff is if we do not store input_objects in txn response, we will need to store raw `TransactionData` b/c of the lossy translation of raw txn to json rpc txn?

Reply:
Agreed that this is convenient, let's add this after March 15th.

> we can store this in indexer but I don't think we should store this in full node, these can be created on the indexer side.

Even on fullnode, we can retrieve these objects using `multi_get_past_objects` right?

Reply:
ya, I thought we have plan to store raw TransactionData?

Reply:
yeah if we had options to get `TransactionData` from FN, it's all good, I meant in case it's only for indexer, then technically having input_objects would also suffice.